### DISCUSSION TITLE
Photoprism: pass smb folder to app

### DISCUSSION THREAD

**Discussion by matrix303** [tteck/Proxmox#91] [Q&A]
Love the photoprism script, super easy to get going and test the app out.

Now I have most of my pictures on a samba share and I have used fstab to mount the pictures share to the LXC. 

Now based on the photoprism documentations, i can simply index my pictures without having to move them. To do this, i have to add make the smb picture folder the original folder as suggested by the volumes section 
 in the documentation. 

https://docs.photoprism.app/getting-started/docker-compose/#volumes

How do i pass through this smb share thats mounted on the lxc container to photoprism?

**Comment by tteck** [comment]
This install method doesn't use volumes

**Reply by matrix303** [reply]
Could you suggest a way to to this? Or would I have to install photoprism manually with a new LXC and docker

**Reply by tteck** [reply]
I'm unsure, maybe someone will chime in that knows 🤞🏻

**Comment by GedasBu** [comment]
Same question from my side. How to add shared folders from my NAS. Methot I have used for Plex container, doesn't work

**Comment by DmitryNefedov** [comment]
Looks like you need to create it using Advanced options and set the container to be privileged. Then you can enable NFS and SMB support for it.
Cannot be changed after container is created.

**Comment by bhavers** [comment]
After enabling SMB support change the originals location in /opt/photoprism/config/.env to point to your photos on the NAS.
